recipes/z3/all/conandata.yml

```diff
@@ -2,7 +2,17 @@ sources:
   "4.8.8":
     url: "https://github.com/Z3Prover/z3/archive/z3-4.8.8.tar.gz"
     sha256: "6962facdcdea287c5eeb1583debe33ee23043144d0e5308344e6a8ee4503bcff"
+  "4.10.2":
+    url: "https://github.com/Z3Prover/z3/archive/z3-4.10.2.tar.gz"
+    sha256: "889fd035b833775c8cd2eb4723eb011bf916a3e9bf08ce66b31c548acee7a321"
 patches:
   "4.8.8":
-    - patch_file: "patches/0001-cmake-use-conan-mpir.patch"
+    - patch_file: "patches/0001-cmake-use-conan-mpir-4.8.8.patch"
```

Comment:
It would be really nice to have https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches-fields so we can track the patches

Reply:
I get this error:
```
[HOOK - conan-center.py] pre_export(): ERROR: [CONANDATA.YML FORMAT (KB-H030)] Additional entries ['patch_description'] not allowed in 'patches':'4.10.2' of conandata.yml (https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H030) 
```

Reply:
My bad, I had to update my hook.